src/main/helm/helm-binary-path.injectable.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { getBinaryName } from "../../common/vars";
+import getAbsolutePathInjectable from "../../common/path/get-absolute-path.injectable";
+import lensResourcesDirInjectable from "../../common/vars/lens-resources-dir.injectable";
+import normalizedPlatformInjectable from "../../common/vars/normalized-platform.injectable";
+import isProductionInjectable from "../../common/vars/is-production.injectable";
+import normalizedPlatformArchitectureInjectable from "../../common/vars/normalized-platform-architecture.injectable";
+
+const helmBinaryPathInjectable = getInjectable({
+  id: "helm-binary-path",
+
+  instantiate: (di) => {
+    const getAbsolutePath = di.inject(getAbsolutePathInjectable);
+    const lensResourcesDir = di.inject(lensResourcesDirInjectable);
+    const normalizedPlatform = di.inject(normalizedPlatformInjectable);
+    const normalizedPlatformArchitecture = di.inject(normalizedPlatformArchitectureInjectable);
+    const isProduction = di.inject(isProductionInjectable);
+
+    const resourcesDir = isProduction
+      ? lensResourcesDir
+      : getAbsolutePath(lensResourcesDir, "binaries", "client", normalizedPlatform);
```

Comment:
Shouldn't this be an injectable on its own? Don't we already have this for `kubectl`?

Reply:
Tried to understand the logic here, but without more context it was hard give good name for those. Also if I'm not mistaken, there is no other usages. It can be split when requirement emerges.

Reply:
There already is uses, just slightly more hidden I guess because of all the DI work lol.

We have this value already as part of `src/common/vars/bundled-resources-dir.injectable.ts` and `src/common/vars/base-bundled-binaries-dir.injectable.ts`

Reply:
I'll figure out this tomorrow. Good catch